server/src/main/java/org/elasticsearch/action/admin/cluster/node/info/NodesInfoRequest.java

```diff
@@ -113,20 +113,6 @@ public void writeTo(StreamOutput out) throws IOException {
         out.writeStringArray(requestedMetrics.toArray(String[]::new));
     }
 
-    /**
-     * Helper method for creating NodesInfoRequests with desired metrics
-     * @param metrics the metrics to include in the request
-     * @return
-     */
-    public static NodesInfoRequest requestWithMetrics(Metric... metrics) {
```

Comment:
This is actually useful - there are several places that could be refactored to use this